app/src/main/res/values/strings.xml

```diff
@@ -181,6 +181,8 @@ How to translate with transifex:
 
     <string name="nc_ignore_battery_optimization_dialog_title">Ignore battery optimization</string>
     <string name="nc_ignore_battery_optimization_dialog_text">Battery optimization is not ignored. This should be changed to make sure that notifications work in the background! Please click OK and select \"All apps\" -> %1$s -> Do not optimize</string>
+    <string name="nc_show_ignore_battery_optimization_hint_title">Show battery optimization hint</string>
+    <string name="nc_show_ignore_battery_optimization_hint_description">When the battery optimization is not ignored, show a hint</string>
```

Comment:
```suggestion
    <string name="nc_show_ignore_battery_optimization_hint_description">When the battery optimization is not disabled or ignored, show a hint</string>
```

Reply:
Don't know about other devices, but mine (Xiaomi) suggests following for 'Battery saver`:
- `No restrictions`: Battery saver doesn't restrict app's activity
- `Battery saver`: ... saver will treat important apps selectively

So I would expect a wording like this:
`When the battery optimization is enabled (not ignored) for this app, show a hint`